ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -898,6 +911,8 @@ spec:
     - name: get-ci-results-attempt
       runAfter:
         - make-index-repo-public
+      when:
```

Comment:
This can't be skipped using a when because the result of this task is used in other places.